docs/developer-docs/latest/guides/auth-request.md

```diff
@@ -116,13 +145,15 @@ The API response contains the **user's JWT** in the `jwt` key.
 }
 ```
 
-You will have to store this `JWT` in your application, it's important because you will have to use it the next requests.
+Save the `JWT` in your application or copy it to your clipboard. You will use it to make future requests. 
 
-### Fetch articles
+::: note
+See the [login documentation](/developer-docs/latest/plugins/users-permissions.md#login) for more information.
+:::
 
-Let's fetch Articles you created.
+### Fetch articles
 
-To do so, you will have to fetch `/articles` route in **GET**.
+Fetch the Articles you created earlier by sending a **GET** request to the `/articles` route:
 
```

Comment:
```suggestion
Sorry @ZPain8464 I was not clear enough here. I think with one more change we can merge this PR. Thanks again! Instead of the screenshot, I was thinking we could add a code block like our API documentation and use the tabs card like the above example. The code for the request is: 
`::: request Example request

`GET http://localhost:1337/api/restaurants`

:::`
```
If you want me to take care of this instead, just let me know. I would like to include this with our 7-September docs release.